web/admin/src/App.css

```diff
@@ -70,32 +118,94 @@ ul li .name {
 }
 
 textarea {
-  height: 5em;
-  width: 40em;
+  font-size: 14px;
+  color: #4A4A4A;
+  height: 100px;	
+  width: 470px;
+  border: 1px solid #AAB9CA;	
+  border-radius: 4px;	
+  background-color: #FFFFFF;	
+  margin-top: 5px;
+  /* box-shadow: inset 0 1px 3px 0 rgba(0,0,0,0.5); */
+  padding: 12px 15px;
 }
 
 button.add {
   font-size: 12px;
-  margin-left: 10px;
-  height: 20px;
-  background-color: #73aaf3;
+  margin: 20px 0px 0px 10px;
+  height: 30px;
+  border-radius: 5px;
+  padding: 0px 10px;
+  background-color: white;
+  border-color: #73aaf3;
+  color: #73aaf3;
+}
+
+button.edit {
+  background-color: white;
+  border: none;
+  font: 300 12px -apple-system, BlinkMacSystemFont, 'Fira Sans', 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
`-apple-system, BlinkMacSystemFont, 'Fira Sans', 'Open Sans', 'Helvetica Neue', sans-serif` is already specified at `:root` so this should not be necessary, right?